src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/EmailConfigurationType.php

```diff
@@ -73,43 +80,60 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('send_emails_to', ChoiceType::class, [
+            ->add(static::FIELD_MAIL_EMAIL_MESSAGE, ChoiceType::class, [
```

Comment:
is there a specific reason why you used `static` instead of `self` here and in the following fields? (I don't think form type classes are meant to be overridden)